### ISSUE TITLE
Schedule should consider timeline for Revision resubmissions

### ISSUE THREAD

**Issue by evansuva** [ieee-security/ongoing-submission-refinement#2]
I think moving to fewer deadlines is a step in the right direction (but 4 is still too many - better to eliminate Q1 altogether). 

I would suggest treating the re-submissions of MR papers differently and using more suitable deadlines for them, though, rather than having them submitted with the same deadlines as regular (new) submissions. I think the way you have the deadlines setup, authors who submit papers for the Sept 1 deadline will get their MR response on Nov 21, and have 9 days to have it ready to submit for their last short at the next year's symposium on Dec 1. This doesn't make a lot of sense to me (or maybe I am misunderstanding things). The point of the MR is to allow for revisions that are more substantial then would be expected within a shepherding timeline, so I think that means more than 9 days (especially over a holiday period).  

One possible fix would be having different deadlines for MR resubmissions than regular papers. I think this would probably help in general, since reviewing them is quite different and better distinguished from reviewing new submissions. Then, the deadline for MR resubmissions (or at least the first opportunity after the MR response) would be ~30+ days from the response, but a separate deadline, say on Jan 3 - no need to allow new submissions on that date, but enough time to actually do a substantial revision but still be reviewed in time for the May conference.

**Comment by alina-oprea** [comment]
Thank you so much for the feedback. 

Of course that 9 days is too little time to prepare a Revision. We had in mind that the authors of a Revised paper can submit to one of the next 2 deadlines after notification. For instance, a paper submitted on March 1 that got the Revised decision on May 21, can be re-submitted to either the June 1 or September 1 deadlines. That gives the authors a little bit more than 3 months to prepare a revision, similarly to the 3-month timeframe we have now. See point (4) from: 

https://github.com/ieee-security/ongoing-submission-refinement/blob/master/refinement-model.md

We will consider the idea to have different deadlines for Revisions, thanks for the suggestion.

**Comment by parno** [comment]
I agree with Dave that 9 days is too short; I think we should disallow or at least heavily discourage Revisions submitted on such a short time scale.  If the changes are that trivial, then it suggests it might not have been a good candidate for Revision.  Perhaps we could allow revisions to be submitted at any time up to the deadline three months after notification, but on the first of each month, batch process any revisions that have been received by that point.  That would allow authors of a revision to get a faster turnaround than the 3 month deadline, but still give the option of taking the full three months.

**Comment by evansuva** [comment]
Thanks Bryan, Alina. I think these make sense, keeping in mind that for authors, there is a big difference between having the paper accepted to S&P 2021 and having to wait until S&P 2022. Whether a paper in S&P 2021 is accepted in March, July, September, or February (etc.) makes little different, although I think in most cases authors would have a slight preference for the earlier acceptances. So the important dates to think about in designing the schedule are the last date when a paper can be submitted that has an MR outcome can still appear in that year's conference. 

With the proposed schedule, if authors who get MR for Q3 are not allowed to submit for Q4 (9 days later), I think this means that the important options are: (1) submit by June 1, to have chance for MR->Accept, or (2) submit on Dec 1 to have chance for direct Accept. (I think this would actually be okay, but could simplify things a lot of this is the intent.)

**Comment by zardus** [comment]
It's not impossible that authors submit to Q3, keep working on the paper, and actually _can_ submit to Q4 within 9 days, although of course it's extremely unlikely.

But anyways, don't we already have a similar thing now, where papers submitted in December or January need a first-shot accept to make it into that year's S&P? The proposed changes make things less granular, but my understanding is that this phenomenon already exists.
